leaf-ui/rappid-extensions/ElementInspector.js

```diff
@@ -1088,6 +1083,14 @@ var ElementInspector = Backbone.View.extend({
       }
 
     }
+    
```

Comment:
In the old function the marked value was not being saved, hope this corrects the old function problem.